### ISSUE TITLE
unnecessary logs

### ISSUE THREAD

**Issue by matanRP** [WireFlare/ngx-slideshow#11]
### Bug Report or Feature Request (mark with an `x`)
```
- [X ] bug report -> please search issues before submitting
- [ ] feature request
```

### Browser and Version?
> Chrome, Firefox, Opera, Edge, etc. Please include the version number

### Framework Versions
Output from: `ng --version`, in case you are using Angular CLI.
Otherwise, output from: `node --version` , `npm --version` and Angular version.

### Repro steps
Click on any of the sliders left or right

### Desired functionality
There are two console.log() that left probably by mistake in the code. is it possible to remove them?

**Comment by matanRP** [comment]
Sorry forgot to mention versions:
npm version : 5.4.2
node version: v8.8.1
 ngx-slideshow: 0.0.10

**Comment by crutchcorn** [comment]
I totally forgot about removing console logs. 😅 I don't have my machine with me currently, so it might be a day or two until I can fix that. Otherwise, if you'd like to make a pull request (the code is pretty small and readable), feel free to!

**Comment by crutchcorn** [comment]
Closed with v0.0.11 released to npm just now
